packages/modals/src/elements/DrawerModal/DrawerModal.tsx

```diff
@@ -187,5 +187,6 @@ DrawerModal.propTypes = {
   restoreFocus: PropTypes.bool,
   onClose: PropTypes.func,
   appendToNode: PropTypes.any,
-  id: PropTypes.string
+  id: PropTypes.string,
+  isOpen: PropTypes.bool
```

Comment:
Does this PR need to be tagged as a `feat(modals): ...` with the addition of this prop?

Reply:
Oops, thanks. I ended up keeping the commits and PR labeled for documentation, but I opened a [new PR](https://github.com/zendeskgarden/react-components/pull/929) to add these prop types.